### PR TITLE
Refractor menu

### PR DESCRIPTION
**Pull request by kistora**
### Description of your PR

Change `Dashboard` to `Mes Services`. Implement a barman role in order to don't display `Mes Services` if the connected user isn't a barman.

### Pre-review TODO

- [x] PR is rebased on top of `origin/master` 
- [ ] PR provide new tests for new behaviors
- [x] PR doesn't introduce commented out code
- [x] PR has been tested locally on the main use cases
- [ ] Documentation have been updated if necessary

### CODE REVIEW

File: client/src/app/_services/auth.service.ts [hunk 1]
```diff
@@ -125,13 +125,15 @@ export class AuthService {
...
                     }));
+                    this.ngxRolesService.addRole('BARMAN', ['']);
                 } else if (connectedUser.specialAccount) {
                     this.$currentUser.next(new ConnectedUser({
                         accountType: 'SpecialAccount',
                         username: connectedUser.specialAccount.connection.username,
                         createdAt: connectedUser.specialAccount.createdAt,
                         specialAccount: connectedUser.specialAccount,
                     }));
+                    this.ngxRolesService.addRole('SPECIAL_ACCOUNT', ['']);
```
Comment (Embraser01) [inline client/src/app/_services/auth.service.ts]: If you add the same role twice, what does it do?

Comment (kistora) [reply client/src/app/_services/auth.service.ts]: I don't know, but why are you asking ?

Comment (Embraser01) [reply client/src/app/_services/auth.service.ts]: Okay nevermind, but why creating a role with empty permissions ?

Comment (kistora) [reply client/src/app/_services/auth.service.ts]: It's just a way to identify the connected account type as a role. 
As a result, I can use it like a role in the menu component.

Comment (Embraser01) [reply client/src/app/_services/auth.service.ts]: Okay right

**Pull request merged**